docs/potato_paper.md

```diff
@@ -0,0 +1,111 @@
+# Potato Paper
+
+## Proof-of-Potato
+
+Proof-of-Potato (PoP) is a protocol for running decentralized sharded off-chain state machines.
+PoP is managed by a potato smart contract hosted on a secure blockchain such as Ethereum.
```

Comment:
Can we say ethereum or private central server?

Reply:
like just "hosted on ethereum"?

Reply:
As in the algorithm needs to be run by centralized server. 
It can be permissionless/trustless like Ethereum, or Ethereum-like chain
Or it can be permissioned/trustful where we just run it from a centralized server that we own/control

Reply:
oh I see. done :D